### PR TITLE
Update Python client instructions

### PR DESCRIPTION
**Pull request by aefhm**
## Description

Add instructions for `web3.py` usage with minimal additional infrastructure.

Devs should select recommended gas strategy as per https://github.com/oasisprotocol/oasis-web3-gateway/issues/501

### CODE REVIEW

File: clients/py/README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ w3.middleware_onion.add(construct_sign_and_send_raw_middleware(account))
 
 # Finally, wrap the provider to add Sapphire end-to-end encryption
 w3 = sapphire.wrap(w3)
+# Optionally, select the gas price oracle implemented in the Oasis Web3 Gateway
```
Comment (matevz) [inline clients/py/README.md:23]:
```suggestion
# Optionally, select the gas price oracle implemented in the Oasis Web3 Gateway.
```

Also, please add punctuation above.

File: clients/py/README.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ w3.middleware_onion.add(construct_sign_and_send_raw_middleware(account))
...
+w3.eth.set_gas_price_strategy(rpc_gas_price_strategy)
```
Comment (matevz) [inline clients/py/README.md:24]: Where is `rpc_gas_price_strategy` defined? Should this be `sapphire.rpc_gas_price_strategy`?

**Review by matevz — CHANGES_REQUESTED** [review]

File: clients/py/README.md (hunk 1 above)
Comment (aefhm) [reply clients/py/README.md:23]: Hvala. I probably should run a commit hook 😬

File: clients/py/README.md (hunk 2 above)
Comment (aefhm) [reply clients/py/README.md:24]: This is from [`web3`](https://github.com/ethereum/web3.py/blob/8f853f5841fd62187bce0c9f17be75627104ca43/web3/gas_strategies/rpc.py#L14). I'll add the import.

**Review by matevz — APPROVED** [review]

**Pull request merged**